bluebottle/bluebottle_drf2/serializers.py

```diff
@@ -40,10 +40,10 @@ def to_internal_value(self, data):
 
 class SorlImageField(RestrictedImageField):
     def __init__(self, geometry_string, **kwargs):
-        self.crop = kwargs.pop('crop', 'center')
```

Comment:
That will still work. It just makes it clear what the signature of the function is. And the code that sets the defaults is now gone.